helpers/match_manipulator.py

```diff
@@ -119,7 +119,8 @@ def updateMerge(self, new_match, old_match, auto_union=True):
 
         auto_union_attrs = [
             "tba_videos",
-            "youtube_videos"
+            "youtube_videos",
+            "vimeo_videos"
```

Comment:
I don't love that this is extending models. I want to try to move to a world where Match videos are also Media objects, and Matches just have a set of Media attached to them.